limitador/Cargo.toml

```diff
@@ -9,10 +9,11 @@ repository = "https://github.com/3scale-labs/limitador"
 documentation = "https://docs.rs/limitador"
 edition = "2018"
 
-# We make redis optional to be able to compile for wasm32.
+# We make redis and infinispan optional to be able to compile for wasm32.
 [features]
-default = ["redis_storage"]
+default = ["redis_storage", "infinispan_storage"]
```

Comment:
I'd say make Redis the default and avoid compiling in Infinispan, because, well, is it expected that users will want to support both at the same time with a single service?

Reply:
The image of the service that we publish needs both, but you're right, the rest of users don't need both.

Reply:
Done in new commit: https://github.com/3scale-labs/limitador/pull/38/commits/1460a7e87c1651f93af2c52a14efe9853cb0e805